Misc/deployment/az_kubes_setup/holorepo-kubes.yaml

```diff
@@ -0,0 +1,270 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: holorepository-accessor
+spec:
+  replicas: 1
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
+  minReadySeconds: 5
+  template:
+    metadata:
+      labels:
+        app: holorepository-accessor
+    spec:
+      nodeSelector:
+        "beta.kubernetes.io/os": linux
+      containers:
+      - name: holorepository-accessor
+        image: holocontainers.azurecr.io/holostorage-accessor:latest
+        imagePullPolicy: Always
+        ports:
+        - containerPort: 3200
+        resources:
+          requests:
+            cpu: 250m
+          limits:
+            cpu: 500m
+        env:
+        - name: AZURE_STORAGE_ACCOUNT
+          value: "holoblobdemo"
+        - name: AZURE_STORAGE_ACCESS_KEY
+          valueFrom:
+            secretKeyRef:
+              name: blobstorekey
+              key: AZURE_STORAGE_ACCESS_KEY
+        - name: ACCESSOR_FHIR_URL
+          value: "https://holostorage-demo.azurewebsites.net"
+        - name: ENABLE_CORS
+          value: "true"
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: holorepository-accessor
+spec:
+  type: LoadBalancer
+  ports:
+  - port: 80
+    targetPort: 3200
+  selector:
+    app: holorepository-accessor
+---
+
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: holorepository-ui-client
+spec:
+  replicas: 1
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
+  minReadySeconds: 5
+  template:
+    metadata:
+      labels:
+        app: holorepository-ui-client
+    spec:
+      nodeSelector:
+        "beta.kubernetes.io/os": linux
+      containers:
+      - name: holorepository-ui-client
+        image: holocontainers.azurecr.io/holorepository-ui-client:latest
+        imagePullPolicy: Always
+        ports:
+        - containerPort: 3000
+        resources:
+          requests:
+            cpu: 250m
+          limits:
+            cpu: 500m
+
+---
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: holorepository-client
```

Comment:
~~I noticed you changed some names. I am fine with the changes, but maybe it would be best to also adjust the docker compose file / service README docker instructions / the name in the buildAndPush step, so all are identical?
(this is optional, feel free to maybe write it on a technical debt list for "future work")~~
Actually, this would require a ton of changes, names of env variables, etc. It's fine actually